### ISSUE TITLE
Change "Move all selected" to "Move selected to..."

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#76]
Originally reported on Google Code with ID 74

```
The "Move all selected" menu item is most likely the simplest/easiest way to move tabs
around, but it's confusing to users.

Some ideas:
* Change the "New Window" button into some sort of... drop-down list doohickey that
would allow you to move selected tabs to either a new window, or into an existing window.

* Add a menu item called "Move to..." that shows a list of the user's windows, including
a "New Window" item. Not as simple as "Move all selected", but possibly less confusing.

Any other ideas?
```

Reported by `pcxunlimited` on 2010-10-06 17:57:24
- **Blocked on**: #76, #77

**Comment by Pauan** [comment]
```
I prefer the 2nd option as it's more obvious. 'Move all selected' doesn't really tell
you much about 'where' - 'Move selected tabs here' would be better but that would widen
the menu.

And, separate menu for selected tabs would be better - with no 'New Tab' or 'Rename
Window' items and the items in 'Selected' submenu (reload, close) merged into the main
menu. But maybe some users would like to right click anywhere and create new tabs etc,
dunno.

About the 'New Window' button, you can add a context menu (right click the button to
show the menu) if you don't want to clutter the UI and keep the option slightly hidden
- only for users that want it (it could be in the faq/doc). Just an idea.
```

Reported by `temp01irc` on 2010-10-06 20:36:32

**Comment by Pauan** [comment]
```
I considered the right-click-on-new-window-button idea, but decided against it because
I want users to be able to move tabs using the keyboard only.

I like the idea of presenting a separate menu when the user right clicks on selected
tabs, but I still need to consider keyboard accessibility.

P.S. I just pushed out a change that implements a "Move selected to..." submenu. I
still need to make it work with the keyboard, though.
```

Reported by `pcxunlimited` on 2010-10-06 20:50:34

**Comment by Pauan** [comment]
```
Also, if you want to try out the change right now, rather than waiting for the 3.1 release,
follow these instructions:

http://code.google.com/p/tab-organizer/wiki/LatestVersion
```

Reported by `pcxunlimited` on 2010-10-06 21:17:44

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-10-06 23:07:17
- **Labels added**: Component-UI

**Comment by Pauan** [comment]
```
Well, this has been fixed. The remaining things to be done are tracked at issue 78 and
issue 79.

P.S. You mentioned session management... feel free to post suggestions on issue 11.
```

Reported by `pcxunlimited` on 2010-10-10 06:48:39
- **Status changed**: `Pending`
- **Blocked on**: #6, #7

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-10-12 03:30:28

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-10-17 17:04:03
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:48
- **Status changed**: `Fixed`
